Scroller/useThemeScroller.js

```diff
@@ -248,7 +248,6 @@ const useThemeScroller = (props) => {
 	delete rest.scrollAndFocusScrollbarButton;
 	delete rest.scrollContainerContainsDangerously;
 	delete rest.scrollContainerHandle;
-	delete rest.scrollContainerRef;
 	delete rest.scrollContentHandle;
```

Comment:
Rather than deleting `scrollContentHandle` from `rest`, what about destructuring it from `props` directly?